js/controller.js

```diff
@@ -501,8 +501,12 @@ module.exports = function(OO, _, $, W) {
       if (!this.state.airPlayStatusIcon) {
         const airPlayState = window.sessionStorage.getItem('airPlayState');
         if (airPlayState === CONSTANTS.AIRPLAY_STATE.CONNECTED) {
-          this.showControlBar();
-          this.state.mainVideoElement.webkitShowPlaybackTargetPicker();
+          const onVideoPlaying = () => {
```

Comment:
A function defined in another function is one more step to the code complexity. No necessity for this here. Let me suggest instead to define in on the same level as others and give a name to it like 'showAirplayPicker'

Reply:
moreover, it's a bit unclear when `toggleAirPlayIcon` adds an event listener while it should just toggle icon

Reply:
Fixed